superset-frontend/cypress-base/cypress/integration/dashboard/index.test.js

```diff
@@ -23,7 +23,7 @@ import DashboardFilterTest from './filter';
 import DashboardLoadTest from './load';
 import DashboardSaveTest from './save';
 import DashboardTabsTest from './tabs';
-import DashboardUrlParamsTest from './url_params';
+import DashboardFormDataTest from './url_params';
```

Comment:
I tried to rename this file to `formData.js`, but a recently added linting rule preferring TypeScript threw a weird exception that I didn't have time to debug now.

Reply:
did you get a screenshot of the exception? It's supposed to post a comment but not block merge if a new js file is added, maybe i missed something though?

Reply:
@etr2460 I don't think it was throwing a normal comment, it looked more like it borked somehow. Let me redo the rename so we can get a screenshot.

Reply:
Here it is: ![image](https://user-images.githubusercontent.com/33317356/83541706-ab427b80-a502-11ea-8c14-00e2dbd56708.png)

Reply:
yikes... so one of the actions the workflow was depending on didn't work :( it doesn't block the PR merge though, right?

Reply:
looks like someone else reported this as well (2 hours ago) https://github.com/trilom/file-changes-action/issues/100. I'll wait a few days to see if it gets resolved, otherwise, i can remove the workflow and try to fix it since it's not required to pass

Reply:
Oh good to know it's a known bug. I can do the rename in a separate PR if we want to debug this further, but for now I'll leave the rename out of this PR.